File: Laboratorio3/Parte_1/Coordenadas.py

x = int(input("Ingrese el valor para X: "))
y = int(input("Ingrese el valor para Y: "))

def cuadrantes(x,y):
    if x > 0 and y > 0:
        print(f"Las coordenadas ({x},{y}) se encuentran en el cuadrante 1")
    elif x > 0 and y < 0:
        print(f"Las coordenadas ({x},{y}) se encuentran en el cuadrante 4")
    elif x < 0 and y < 0:
        print(f"Las coordenadas ({x},{y}) se encuentran en el cuadrante 3")
    elif x < 0 and y > 0:
        print(f"Las coordenadas ({x},{y}) se encuentran en el cuadrante 2")
    else:
        print("Las coordenadas son en el origen (0,0)")

if __name__ == '__main__':
    cuadrantes(x,y)

File: Lab2/Parte_1/Coordenadas.java
import java.util.Scanner;
class Coordenadas{
    public static void main(String [] args){
        try (Scanner myObj = new Scanner(System.in)) 
        {
            int x, y;
            System.out.println("Ingrese el valor para X:");
            x = myObj.nextInt();
            System.out.println("Ingrese el valor para Y:");
            y = myObj.nextInt();
            if(x > 0 && y >0)
            {
                System.out.println("Las coordenadas ("+ x +","+ y +") se encuentran en el cuadrante 1");
            }
            else if(x > 0 && y < 0 )
            {
                System.out.println("Las coordenadas ("+ x +","+ y +") se encuentran en el cuadrante 4");
            }
            else if(x < 0 && y < 0 )
            {
                System.out.println("Las coordenadas ("+ x +","+ y +") se encuentran en el cuadrante 3");
            }
            else if(x < 0 && y > 0 )
            {
                System.out.println("Las coordenadas ("+ x +","+ y +") se encuentran en el cuadrante 2");
            }
            else
            {
                System.out.println("Las coordenadas son el origen (0,0)");
            }
        }
    }
}